client-app/ui-kit/components/molecules/select/vc-select.vue

```diff
@@ -1,143 +1,125 @@
 <template>
```

Comment:
This component is too much complex, I think we need to find a way to separate it intomultiple components (single/multiple, container/filter/single/multi, etc.)

Reply:
Yes, we can, we need to talk about it and create a new ticket.

Reply:
https://virtocommerce.atlassian.net/browse/VCST-471